### DISCUSSION TITLE
Newbie: Making a bash script to transcode multiple files

### DISCUSSION THREAD

**Discussion by thepragmaticmero** [gianni-rosato/svt-av1-psy#86] [General]
I'm kinda new to av1 encoding, I don't know what to use and the documentation is kinda confusing to me. I just need a cli way to transcode a video, I'll take care of writing the bash script on my own, I just need a command with this structure.
`encode --input=/path/to/input --output=/path/to/output [recommended encoding features with xpsnr for best encoding]` 
My goal is kinda to encode achieving the least amount of disk space. Not too compressed obviously, but prioritizing disk space.
I've read the documentation kind of quickly and saw a couple of parameters, the thing is, I don't know where to add them. Are those commands for `ffmpeg`? is `SvtAv1EncApp` useful for my use case?

**Comment by thepragmaticmero** [comment]
Ok I found out using `SvtAv1EncApp --help` that it has the Psychovisual options (or as the project calls it: pussy). Still, what's the recommended way of doing it? I don't wanna do it wrong Q-Q I need a tiny bit of advice

**Comment by baysonfox** [comment]
[Using SVT-AV1 within ffmpeg](https://gitlab.com/AOMediaCodec/SVT-AV1/-/blob/master/Docs/Ffmpeg.md) from upstream is probably  the thing you would be interested in.

**Reply by thepragmaticmero** [reply]
I guess I just need to add the parameters that pussy adds to SVT-AV1, right? Thanks for the help and the link~

**Comment by Uranite** [comment]
You can use my build of ffmpeg that has svt-av1-psy
https://github.com/Uranite/FFmpeg-Builds/releases/tag/latest
Btw, its called psy and not pussy 😉
